### DISCUSSION TITLE
网格线颜色

### DISCUSSION THREAD

**Discussion by leon9508** [antvis/G2#5396] [Q&A] [answered]
- **G2 Version**: 4.0.8
- **Platform**:
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:  https://codesandbox.io/s/silly-tereshkova-k9dkzg?file=/index.ts

水平网格线的颜色与设置的网格线的颜色不符合，且通过更改透明度的方式也不生效：

<img width="1422" alt="image" src="https://github.com/antvis/G2/assets/25746834/d758ab0a-7ff0-4f08-a3d8-de53c6f0d08b">

**Comment by hustcc** [comment, marked as answer]
![image](https://github.com/antvis/G2/assets/7856674/6a29ec7d-c7bd-45da-95bb-e3cc186978e4)


可以生效。

**Reply by leon9508** [reply]
没看出来有色差吗， grid设置的是red红色，但是很明显有一定透明度，和左侧label设置为red表现不一致
<img width="1035" alt="image" src="https://github.com/antvis/G2/assets/25746834/54da6394-c55e-478d-8249-9475c4cc504c">
